code/components/jomjol_controlGPIO/server_GPIO.cpp

```diff
@@ -439,7 +441,7 @@ void GpioHandler::registerGpioUri()
     httpd_uri_t camuri = { };
     camuri.method    = HTTP_GET;
     camuri.uri       = "/GPIO";
-    camuri.handler   = callHandleHttpRequest;
+    camuri.handler   = APPLY_BASIC_AUTH_FILTER(callHandleHttpRequest);
```

Comment:
Are you sure this works as intended? `callHandleHttpRequest` is a function pointer here. I couldn't find anything about `APPLY_BASIC_AUTH_FILTER`, so not sure what this does ...